web/vibe/web/rest.d

```diff
@@ -469,23 +469,24 @@ URLRouter registerRestInterface(TImpl)(URLRouter router, TImpl instance, string
 HTTPServerRequestDelegate serveRestJSClient(I)(RestInterfaceSettings settings)
 	if (is(I == interface))
 {
+	import std.datetime.systime : SysTime;
 	import std.digest.md : md5Of;
 	import std.digest : toHexString;
 	import std.array : appender;
 
+	import vibe.http.fileserver : ETag, handleCache;
+
 	auto app = appender!string();
 	generateRestJSClient!I(app, settings);
```

Comment:
Unrelated, but shouldn't we cache this?
It seems pretty silly to regenerate the entire JS client on _every_ request (the interface won't change during the application's execution), so the cache just needs to check that settings didn't change.

Reply:
we should create that as separate issue if it's planned to change that